bubuku/broker.py

```diff
@@ -24,8 +25,42 @@ def is_running_and_registered(self):
         return self.id_manager.is_registered()
 
     def stop_kafka_process(self):
+        """
+        Stops kafka process (if it is running) and says if this topic is still is a leader on in ISR list for some
+        partitions
+        :return: True, if broker is stopped and is not a leader or a isr.
+        """
         self._terminate_process()
         self._wait_for_zk_absence()
+        return not self._have_leadership()
+
+    def _is_clean_election(self):
```

Comment:
ok, fine for me. However I think that "unclean election" doesn't really have negation.